docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,717 @@
+The payload of the Transaction that creates RecordTypes is the same as
+the RecordType object itself: it has a name and a list of Properties.
+
+.. code-block:: protobuf
+
+   message CreateRecordTypeAction {
+       string name = 1;
+
+       repeated PropertySchema properties = 2;
+   }
+
+
+A CreateRecordType transaction is invalid if
+
+- the signer is not registered as an Agent,
+- its list of Properties is empty,
+- the name of the RecordType is the empty string, or
+- a RecordType with its name already exists.
+
+
+Update Properties
+-----------------
+
+An UpdateProperties transaction contains a record_id and a list of
+PropertyValues (see Create Record above). It can only be (validly)
+sent by an Agent authorized to report on the Property.
+
+.. code-block:: protobuf
+
+   message UpdatePropertiesAction {
+       // The natural key of the Record
+       string record_id = 1;
+
+       repeated PropertyValue properties = 2;
+   }
+
+
+An UpdateProperties transaction is invalid if
+
+- the Record does not exist,
+- the Record is final,
+- its signer is not authorized to report on that Record, or
+- if any of the provided PropertyValues do not match the types
+  specified in the Record's RecordType.
+
+
+Create Proposal
+---------------
+
+A CreateProposal transaction creates an open Proposal concerning some
+Record from the signer to the receiving Agent. This Proposal can be
+for transfer of ownership, transfer of custodianship, or authorization
+to report. If it is a reporter authorization Proposal, a nonempty list
+of Property names must be included.
+
+.. code-block:: protobuf
+
+   message CreateProposalPayload {
+       enum Role {
+           OWNER = 1;
+           CUSTODIAN = 2;
+           REPORTER = 3;
+       }
+
+       string record_id = 1;
+
+       // The public key of the Agent to whom the Proposal is sent
+       // (must be different from the Agent sending the Proposal).
+       string receiving_agent = 3;
+
+       repeated string properties = 4;
+
+       Role role = 5;
+   }
+
+
+A CreateProposal transaction will be invalid if
+
+- the signer is not the owner and the Proposal is for transfer of
+  ownership or reporter authorization,
+- the signer is not the custodian and the Proposal is for transfer of
+  custodianship,
+- the receiving Agent is not registered (the signer must be registered
+  as well, but this is implied by the previous two conditions),
+- there is already an open Proposal for the Record and receiving Agent
+  for the specified role,
+- the Record is final, or
+- the Proposal is for reporter authorization and the list of Property
+  names is empty.
+
+
+Answer Proposal
+---------------
+
+An Agent who is the receiving Agent for a Proposal for some Record can
+accept or reject that Proposal, marking the Proposal's status as
+`accepted` or `rejected`. The Proposal's issuing_agent cannot accept
+the Proposal, but can reject it. This will mark the Proposal's status
+as `cancelled` rather than `rejected`.
```

Comment:
> The Proposal's issuing_agent cannot accept the Proposal, but can reject it.

I don't think this is actually accurate, since the issuing_agent would never use the `REJECT` enum. I suggest replacing the last two sentences with this one:

```
The Proposal's ``issuing_agent`` cannot accept or reject it, but
can cancel it.
```

Reply:
Also, backticks.